app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from datetime import date
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+
+        expiration_date = visitor["vaccine"]["expiration_date"]
```

Comment:
Use method get to avoid KeyError
```suggestion
        expiration_date = visitor["vaccine"].get("expiration_date")
```

Reply:
Ok. Thanks. You've given me a very useful advice. But I wouldn't say I like dictionaries in Python at all. JS objects are much better and convenient in use)